account-service/src/main/java/org/bfsi/account/controller/AccountController.java

```diff
@@ -12,59 +12,46 @@
 
 @CrossOrigin
 @RestController
-@RequestMapping("/api/v1/accounts/")
+@RequestMapping("/api/v1/accounts")
 @Slf4j
 public class AccountController {
 
     @Autowired
     private AccountService accountService;
 
-    @GetMapping("health")
+    @GetMapping("/health")
     public ResponseEntity<String> health() {
         log.info("Info level log message");
         log.debug("Debug level log message");
         log.error("Error level log message");
         return new ResponseEntity("OK", HttpStatus.OK);
-
     }
-    @GetMapping("account")
-    public ResponseEntity<List<AccountEntity>> getAccountEntities() {
 
+    @GetMapping
+    public ResponseEntity<List<AccountEntity>> getAccountEntities() {
```

Comment:
@margi212 entity is directly exposed, we can use DTO for account object. Please change it in next commit.